nextflow.config

```diff
@@ -16,6 +16,7 @@ params {
 
     // Reference genome options
     genome                     = null
+    igenomes_base              = 's3://ngi-igenomes/igenomes/'
```

Comment:
I'm not sure about the usage of igenomes with this pipeline, I think I remember that we were not using it